pkg/api/types.go

```diff
@@ -962,10 +962,13 @@ type ContainerStatus struct {
 	Ready bool `json:"ready"`
 	// Note that this is calculated from dead containers.  But those containers are subject to
 	// garbage collection.  This value will get capped at 5 by GC.
-	RestartCount int    `json:"restartCount"`
-	Image        string `json:"image"`
-	ImageID      string `json:"imageID"`
-	ContainerID  string `json:"containerID,omitempty"`
+	RestartCount      int              `json:"restartCount"`
+	ContainerID       string           `json:"containerID,omitempty"`
+	Image             string           `json:"image"`
+	ImageID           string           `json:"imageID"`
```

Comment:
This ID is completely dependent on the runtime. Our APIs try to be agnostic to runtimes. Why do we need this? As I said in the previous PR, cAdvisor already exposes docker image information.